client/tests/integration/domain_owner_permissions.rs

```diff
@@ -58,7 +59,7 @@ fn domain_owner_domain_permissions() -> Result<()> {
 
     // check that "alice@wonderland" as owner of domain can edit metadata in her domain
     let key: Name = "key".parse()?;
-    let value: Name = "value".parse()?;
+    let value = JsonString::new("value");
     test_client.submit_blocking(SetKeyValue::domain(kingdom_id.clone(), key.clone(), value))?;
```

Comment:
```suggestion
    let value = "value";
    test_client.submit_blocking(SetKeyValue::domain(kingdom_id.clone(), key.clone(), value))?;
```

I think we should implement `From<&str> for JsonString` and have it just like this

Reply:
Integration tests will be refactored here

Will be done here: https://github.com/hyperledger/iroha/issues/4749